### ISSUE TITLE
[ErrorException] Array to string conversion 

### ISSUE THREAD

**Issue by jp1971** [phase2/octane#2]
Running `composer install` or `composer update` throws `[ErrorException] Array to string conversion` during `Applying patches for drupal/core`.

**Comment by mike-potter** [comment]
Can you show more context for this error message by pasting part of the console output you get (+-10 lines around the error or so)?

There _is_ currently an issue applying the patch "https://www.drupal.org/files/issues/make_inherited_install-1356276-129.patch".  I just get "Could not apply patch! Skipping." so maybe your error is giving more detail.

We haven't tracked down the cause of this patch problem yet.  It applies just fine when done manually, so we need to debug the composer-patch process for that.

**Comment by jp1971** [comment]
@mike-potter Sorry for the delayed response. Here's the output you requested:

```
- Installing symfony-cmf/routing (1.3.0)
Loading from cache

- Installing drupal/core (8.1.8)
Downloading: 100%         

- Applying patches for drupal/core
https://www.drupal.org/files/issues/make_inherited_install-1356276-129.patch (2743197 - Provide a workaroud to load additional profiles via settings.php)
Could not apply patch! Skipping.

[ErrorException]            
Array to string conversion  
```

**Comment by mike-potter** [comment]
Ahh, that makes sense.  That issue had trouble applying the patch because of mismatched comments.  The thread was closed (#1356276) in favor of the latest patch here: https://www.drupal.org/node/1356276#comment-11479133 which is what the latest version of Octane is using.  This patch should apply to 8.1.x, 8.2.x, and 8.3.x.
